website/docs/gettingStarted/quickstart.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Quickstart
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+This document gives you a quick view on the basic usage of Submarine platform. You can finish each steps of ML model lifecycle on the platform without messing up with the troublesome environment problems.
+
+## Installation
+
+### Prepare a Kubernetes cluster
+
+1. Prerequisite
+
+- Check dependency page for the compatible version
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [helm](https://helm.sh/docs/intro/install/) (Helm v3 is minimum requirement.)
+- [minikube](https://minikube.sigs.k8s.io/docs/start/).
+
+2. Start minikube cluster
+```
+$ minikube start --vm-driver=docker --cpus 8 --memory 4096 --kubernetes-version v1.15.11
+```
+
+### Launch submarine in the cluster
+
+1. Clone the project
+```
+$ git clone https://github.com/apache/submarine.git
+```
+
+2. Install the resources by helm chart
+```
+$ cd submarine
+$ helm install submarine ./helm-charts/submarine
+```
+
```

Comment:
In my opinion, it is necessary to tell users that how to connect to the Submarine workbench.

https://github.com/apache/submarine/blob/master/website/docs/gettingStarted/localDeployment.md#access-submarine-in-a-cluster